src/CacheManager.Serialization.ProtoBuf/project.json

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "0.9.0-*",
+  "description": "CacheManager extension package providing ProtoBuf serialization for distributed caches. CacheManager is an open source caching abstraction layer for .NET written in C#. It supports various cache providers and implements many advanced features, the ProtoBuf serializer can be used in place of the default Binary Serializer",
+  "packOptions": {
+    "licenseUrl": "https://github.com/MichaCo/CacheManager/blob/master/LICENSE",
+    "projectUrl": "http://www.cachemanager.net",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/MichaCo/CacheManager"
+    },
+    "summary": "CacheManager extension package providing ProtoBuf serialization for distributed caches.",
+    "tags": [ "Caching", "Cache", "CacheManager", "Distributed Cache", "StackExchange Redis", "Azure AppFabric", "Memcached" ]
+  },
+  "authors": [ "Wenisman" ],
+  "buildOptions": {
+    "allowUnsafe": false,
+    "languageVersion": "csharp6",
+    "warningsAsErrors": true
+  },
+  "copyright": "Copyright (c) 2015 MichaConrad",
+  "configurations": {
+    "Debug": {
+      "buildOptions": {
+        "define": [ "DEBUG" ]
+      }
+    },
+    "Release": {
+      "buildOptions": {
+        "define": [ "RELEASE" ],
+        "optimize": true
+      }
+    }
+  },
+  "dependencies": {
+    "CacheManager.Core": "0.9.0-*",
+    "NETStandard.Library": "1.6.0",
+    "protobuf-net": "2.1.0"
+  },
+
+  "frameworks": {
+    "netstandard1.6": {
+      "imports": "dnxcore50"
```

Comment:
always use the lowest possible netstandard. netstandard1.6 is .net4.6.x compatible only...
netstandard1.3 is supported by protobuf-net